src/main/kotlin/com/sourcegraph/cody/chat/ui/LlmDropdown.kt

```diff
@@ -65,6 +64,8 @@ class LlmDropdown(
 
     isEnabled = chatModelProviderFromState == null
     isVisible = !isEnterpriseAccount
+    setMaximumRowCount(15)
```

Comment:
I think we want to make all models visible at once so free users can easier notice pro models at the bottom (so they have some motivation to convert)